openqa-mail-notification/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.opensuse.org/home/vpereirabr/dockerimages/containers/obs-tools/base:latest
+
+RUN zypper --non-interactive --quiet ref \
+ && zypper -n in ruby2.5-rubygem-mail
```

Comment:
I suggest using long option and command names, so it would be `refresh` instead of `ref` and on the line 4, it would be `zypper --non-interactive install`.